src/Layer.js

```diff
@@ -179,7 +181,7 @@ export default class Layer {
 
         await this._context;
         if (this._atomicChangeUID > uid) {
-            throw new Error('Another atomic change was done before this one committed');
+            throw new CartoRuntimeError('Another atomic change was done before this one committed (1)');
```

Comment:
I would remove `(1)` and `(2)` even the error message is the same. I recognise I do not know if this is a convention, if it is go ahead. The rest looks ✨

Reply:
I think the are useless now... But, if in the future we discover or introduce a bug, they could be highly useful. And this part of the code has been historically error-prone. I'd leave them.